DRODLib/CurrentGame.cpp

```diff
@@ -68,6 +68,12 @@
 
 #define NO_CHECKPOINT (static_cast<UINT>(-1))
 
+//Max weather values. One less than defines in DROD/RoomWidget.h
+#define MAX_ROOM_LIGHT (6)
```

Comment:
I'll approve and merge this PR, but for ease of future maintainability, I'd suggest placing these defines in CurrentGame.h and having the defines in DROD/RoomWidget.h based off of these values so they only need to be changed in one place, and so future changes to RoomWidget.h don't cause bugs (iirc, those could be changed with impunity up to this point, if we ever wanted to).